adlfs/spec.py

```diff
@@ -398,7 +398,11 @@ def _strip_protocol(cls, path: str):
 
         STORE_SUFFIX = ".dfs.core.windows.net"
         logger.debug(f"_strip_protocol for {path}")
-        if not path.startswith(("abfs://", "az://", "abfss://")):
+        if isinstance(cls.protocol, str):
+            protocol_startswith = (f"{cls.protocol}://",)
+        else:
+            protocol_startswith = tuple([f"{proto}://" for proto in cls.protocol])
```

Comment:
(nit) this makes a list and then a tuple.
```suggestion
            protocol_startswith = tuple(f"{proto}://" for proto in cls.protocol)
```